### ISSUE TITLE
Loading scene causes octree shift -y if Y clip>0

### ISSUE THREAD

**Issue by jackjt8** [chunky-dev/chunky#801]
**Initial scene**
`Y min clip` = 4; `Camera Position Y`=7.5
![image](https://user-images.githubusercontent.com/3740917/106198322-cfe31600-61ab-11eb-8232-24a73b3fbb18.png)

---

**Saving and loading the scene**
![image](https://user-images.githubusercontent.com/3740917/106198390-e2f5e600-61ab-11eb-9d1d-4abf8935b191.png)

Noted that Octree has been shifted by Y-=2; X,Z unchanged.
![image](https://user-images.githubusercontent.com/3740917/106198476-fbfe9700-61ab-11eb-99a3-3568740854d7.png)

---

Test2:

`Y min clip` = 60; `Camera Position Y`=79.5
Upon scene load octree shifted to Y=49.5; ie -30

`Y max clip` = 156 (-100); `Camera Position Y`=79.5
Upon scene load octree shifted to Y=29.5; ie -50

---

Hitting `Reload Chunks` corrects the octree's Y position.

tl;dr For Y min/max clip > 0 offsets octrees position upon scene loading by `Y -= [Y min/max clip]/2` (ignoring cases of both min/max being set)

**Comment by jackjt8** [comment]
Octree is not shifted in 2.3.0-49-g08485190 but is by 2.3.0-52-gd0476da5 --- https://github.com/chunky-dev/chunky/commit/a722b55ee7530ba149aad6211486b45d606e90b8
